client/wc-api/customers/selectors.js

```diff
@@ -0,0 +1,43 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { isNil } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getResourceName } from '../utils';
+import { DEFAULT_REQUIREMENT } from '../constants';
+
+const getCustomers = ( getResource, requireResource ) => (
+	query = {},
+	requirement = DEFAULT_REQUIREMENT
+) => {
+	const resourceName = getResourceName( 'customers-query', query );
+	const ids = requireResource( requirement, resourceName ).data || [];
+	return ids.map( id => getResource( getResourceName( 'customer', id ) ).data || {} );
+};
+
+const isGetCustomersRequesting = getResource => ( query = {} ) => {
+	const resourceName = getResourceName( 'customers-query', query );
+	const { lastRequested, lastReceived } = getResource( resourceName );
+
+	if ( isNil( lastRequested ) || isNil( lastReceived ) ) {
+		return true;
+	}
+
+	return lastRequested > lastReceived;
+};
+
+const isGetCustomersError = getResource => ( query = {} ) => {
+	const resourceName = getResourceName( 'customers-query', query );
+	return getResource( resourceName ).error;
```

Comment:
Oh, right, that's a good point, I was convinced `error` was a boolean instead of an object.

I fixed it here for now (09d7eafc5ac172b03e05a2f45fa0b5173c214e8f and 12d2550f4ce40613df06bca6fd3d5fc3799a2527) and created a PR with fixes for other selectors that were also affected by this: #1101.